app/util/path-builder.ts

```diff
@@ -119,8 +115,6 @@ export const pb = {
   ipPoolEdit: (params: IpPool) => `${pb.ipPool(params)}/edit`,
   ipPoolRangeAdd: (params: IpPool) => `${pb.ipPool(params)}/ranges-add`,
 
-  inventory: () => '/system/inventory',
-  rackInventory: () => '/system/inventory/racks',
```

Comment:
The deleted ones in here were never used. Some were used to construct other paths but never used directly.